src/schema/v2/home/__tests__/home_page_artwork_modules.test.js

```diff
@@ -355,4 +356,51 @@ describe("HomePageArtworkModules", () => {
       expect(keys).toEqual(["followed_galleries"])
     })
   })
+
+  it("doesn't return fairs that haven't opened yet", async () => {
```

Comment:
**suggestion:** It may help to have a test for the case where there are no running fairs.